fix dependency on event-stream

[gulp-typedoc@2.2.0 depends on event-stream^3.3.4](https://github.com/rogierschouten/gulp-typedoc/blob/v2.2.0/package.json#L27), so when users install gulp-typedoc, they'll get the highest version of v3. [event-stream 3.3.5 through 3.3.6](https://github.com/dominictarr/event-stream/commit/e3163361fed01384c986b9b4c18feb1fc42b8285#diff-b9cfc7f2cdf78a7f4b91a753d10865a2) depends on a package called "flatmap-stream", which was once on the npm registry, but [it is no longer there](https://www.npmjs.com/package/flatmap-stream).

So when users install gulp-typedoc, they'll get the following error:

> npm ERR! No valid versions available for flatmap-stream

event-stream@3.3.4 doesn't depend on flatmap-stream, and neither does v4.0.0 or higher. Possible ways to fix this:

1. lock the dependency of `"event-stream": "^3.3.4"` to `"event-stream": "3.3.4"`, or
2. update to `"event-stream": "^4.0.0"` (breaking change)

Hi, thank you for reporting!

I'm in the middle of moving house, could you maybe send a PR my way? 

On November 26, 2018 7:15:57 PM GMT+01:00, Chris Harvey <notifications@github.com> wrote:
>[gulp-typedoc@2.2.0 depends on
>event-stream^3.3.4](https://github.com/rogierschouten/gulp-typedoc/blob/v2.2.0/package.json#L27),
>so when users install gulp-typedoc, they'll get the highest version of
>v3. [event-stream 3.3.5 through
>3.3.6](https://github.com/dominictarr/event-stream/commit/e3163361fed01384c986b9b4c18feb1fc42b8285#diff-b9cfc7f2cdf78a7f4b91a753d10865a2)
>depends on a package called "flatmap-stream", which was once on the npm
>registry, but [it is no longer
>there](https://www.npmjs.com/package/flatmap-stream).
>
>So when users install gulp-typedoc, they'll get the following error:
>
>> npm ERR! No valid versions available for flatmap-stream
>
>event-stream@3.3.4 doesn't depend on flatmap-stream, and neither does
>v4.0.0 or higher. Possible ways to fix this:
>
>1. lock the dependency of `"event-stream": "^3.3.4"` to
>`"event-stream": "3.3.4"`, or
>2. update to `"event-stream": "^4.0.0"` (breaking change)
>
>-- 
>You are receiving this because you are subscribed to this thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/rogierschouten/gulp-typedoc/issues/25

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

--- a/package.json
+++ b/package.json
@@ -24,7 +24,7 @@
   "license": "ISC",
   "dependencies": {
     "ansi-colors": "^1.0.1",
-    "event-stream": "^3.3.4",
+    "event-stream": "3.3.4",
     "fancy-log": "^1.3.2",
     "plugin-error": "^0.1.2"
   },

